crates/control-interface/src/types.rs

```diff
@@ -130,6 +132,15 @@ pub struct Host {
     pub version: Option<String>,
 }
 
+// TODO(pre-1.0): remove me once all clients are updated to control interface v0.33
+impl Host {
+    #[must_use] pub fn lattice(&self) -> Option<&String> {
```

Comment:
Why the `must_use` here?

Reply:
No idea, clippy fix automatically added this